.github/workflows/ci.yml

```diff
@@ -55,14 +60,21 @@ jobs:
             - name: Setup Python
               uses: actions/setup-python@v5
 
+            # Needed within cibuildwheel
+            - name: Install uv
+              run: curl -LsSf https://astral.sh/uv/install.sh | sh
+
             - uses: pypa/cibuildwheel@v2.20.0
               env:
+                CIBW_BUILD_FRONTEND: "build[uv]"
                 CIBW_BUILD: "${{ matrix.build }}${{ matrix.python }}*"
                 CIBW_ARCHS: ${{ matrix.arch }}
+                CIBW_TEST_EXTRAS: "test"
```

Comment:
Actually this can go now, since we added the `[test]` extra to dev dependencies.

Reply:
I was wrong! But now just got rid of pytest to simplify dependencies.